server-mock/server/minipool-deposit-signature.go

```diff
@@ -1,13 +1,52 @@
 package server
 
 import (
+	"encoding/json"
+	"fmt"
 	"net/http"
 
+	"github.com/ethereum/go-ethereum/crypto"
 	apiv2 "github.com/nodeset-org/nodeset-client-go/api-v2"
+	"github.com/nodeset-org/nodeset-client-go/server-mock/db"
+	nsutil "github.com/nodeset-org/nodeset-client-go/utils"
+	"github.com/rocket-pool/node-manager-core/utils"
 )
 
 func (s *NodeSetMockServer) minipoolDepositSignature(w http.ResponseWriter, r *http.Request) {
+	var request struct {
```

Comment:
This is ok for now but once we build the client routes we're going to end up moving this type into the client and reusing that type here.

Reply:
ok